src/api/reservationsApi.js

```diff
@@ -0,0 +1,19 @@
+import { APP_ID } from '../config.js';
+
+const baseUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${APP_ID}/reservations`;
+
+export const addReservations = (itemId, userR, dateStart, dateEnd) => fetch(baseUrl, {
+  method: 'POST',
+  headers: { 'Content-Type': 'application/json' },
+  body: JSON.stringify({
+    item_id: itemId,
+    username: userR,
+    date_start: dateStart,
+    date_end: dateEnd,
+  }),
```

Comment:
Record persistence with the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) well implemented! 👌